### ISSUE TITLE
Release 1.1.1 problems

### ISSUE THREAD

**Issue by mshick** [LoicMahieu/react-styled-flexboxgrid#38]
It immediately triggers the following error:

```
app.52fee47….js:781 TypeError: Cannot read property 'className' of undefined
    at eval (webpack:///./~/react-styled-flexboxgrid/lib/createProps.js?:11:52)
    at Array.filter (native)
    at createProps (webpack:///./~/react-styled-flexboxgrid/lib/createProps.js?:10:22)
    at Row (webpack:///./~/react-styled-flexboxgrid/lib/components/Row.js?:46:90)
    at eval (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:306:16)
    at measureLifeCyclePerf (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:75:12)
    at ReactCompositeComponentWrapper._constructComponentWithoutOwner (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:305:14)
    at ReactCompositeComponentWrapper._constructComponent (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:280:21)
    at ReactCompositeComponentWrapper.mountComponent (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:188:21)
    at Object.mountComponent (webpack:///./~/react-dom/lib/ReactReconciler.js?:46:35)
```

I reverted to 1.1.0 and things are working properly.

**Comment by stevebluck** [comment]
I get the same thing.

**Comment by LoicMahieu** [comment]
Hi guys,
Stupid error...
Fixed in 1.1.2

Thanks

**Comment by mshick** [comment]
Fixed it for me. Thanks!
